source/operations/install-deploy-manage/migrate-fs-gateway.rst

```diff
@@ -73,128 +72,143 @@ Procedure
    - Replace ``PATH`` with the IP address or hostname and port for the new deployment.
    - Replace ``ACCESSKEY`` and ``SECRETKEY`` with the credentials you used when creating the new deployment.
 
-#. Export the existing deployment's **configurations**
+#. Migrate settings according to the type of deployment:
 
-   Use the :mc-cmd:`mc admin config export <mc admin config export>` export command with the existing MinIO Client to retrieve the configurations defined for the existing standalone MinIO deployment.
+   .. tab-set::
 
-   .. code-block:: shell
-      :class: copyable
+      .. tab-item:: Gateway
 
-      mc admin config export ALIAS > config.txt
+         a. Migrate environment variables
 
-   - Use the existing MinIO Client.
-   - Replace ``ALIAS`` with the alias used for the existing standalone deployment you are retrieving values from. 
+            Copy the :ref:`environment variables <minio-server-environment-variables>` from the existing deployment's ``/etc/default/minio`` file to the same file in the new deployment.
+            You may omit any ``MINIO_CACHE_*`` and ``MINIO_GATEWAY_SSE`` environment variables, as these are no longer used.
 
-#. Import **configurations** from existing standalone deployment to new deployment with the new MinIO Client
+      .. tab-item:: Filesystem mode
```

Comment:
My understanding is the filesystem mode instructions are (mostly) ok. I haven't tested them yet, the old version of the install docs reference instructions on the download site that no longer exist so it's confusing.

Reply:
Whatever we do now is better than it was, so deal with that if/when it comes up.